main.js

```diff
@@ -61,12 +62,22 @@ function hide(element) {
 
 function receiveMessage() {
   if (affirmationButton.checked) {
-      message.innerText = affirmations[getRandomIndex(affirmations)];
-      hide(sectionImage);
-      display(message);
+    message.innerText = affirmations[getRandomIndex(affirmations)];
+    hide(sectionImage);
+    display(message);
+    display(clearMessageButton);
   } else if (mantraButton.checked) {
-      message.innerText = mantras[getRandomIndex(mantras)];
-      hide(sectionImage);
-      display(message);
+    message.innerText = mantras[getRandomIndex(mantras)];
+    hide(sectionImage);
+    display(message);
+    display(clearMessageButton);
+  } else {
+    alert('Please select your type of message.');
   }
 };
+
+function clearMessage() {
```

Comment:
I was not able to test your code because the pull request is closed, but I would just check that your error handling alert still works after clicking the clear button. If it doesn't, you might need to reset the form in the clearMessage function.